src/lecturer/components/createSessionView/CreateSessionView.tsx

```diff
@@ -97,6 +117,7 @@ export function CreateSessionView() {
                     />
                 )}
             </div>
+            <input type="file" name="file" onChange={(e) => onChange(e)}/>
         </div>
-    );
-}
+    ); // tego <input type file ... też
```

Comment:
To czemu jest?

Reply:
żeby testować czy dobrze mechanizm działa i przechować gdzieś kod